stdlib/nodejs/require.rb

```diff
@@ -16,15 +16,15 @@ def __prepare_require__(path)
     end
 
     name
-  rescue => e
-    raise [path, name, full_path].inspect+e.message
+  rescue StandardError => e
```

Comment:
looked up in the style guide and didn't find it enforced, let alone a good reason to do that, can the rule be disabled if present?